app/code/Magento/InventoryCatalog/Model/GetStockIdForCurrentWebsite.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalog\Model;
+
+use Magento\InventorySalesApi\Api\Data\SalesChannelInterface;
+use Magento\InventorySalesApi\Api\StockResolverInterface;
+use Magento\Store\Model\StoreManagerInterface;
+
+/**
+ * @inheritdoc
+ */
+class GetStockIdForCurrentWebsite implements GetStockIdForCurrentWebsiteInterface
```

Comment:
Looks like we use this code only in `InventoryCatalog` so we don't need this interface
Because if we introduce interface we need to support it

Class for avoiding boilerplate code is ok but looks like interface is redundant

Reply:
Done.